server/views/pages/prisons/categoryGroups/addCategoryGroup.njk

```diff
@@ -0,0 +1,95 @@
+{% extends "partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
+{% from "moj/components/page-header-actions/macro.njk" import mojPageHeaderActions %}
+{%- from "govuk/components/button/macro.njk" import govukButton -%}
+{%- from "govuk/components/input/macro.njk" import govukInput -%}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+{% set pageHeaderTitle = 'Add a prisoner category group' %}
+{% set pageTitle = applicationName + " - Prisoner category groups - " + pageHeaderTitle %}
```

Comment:
```suggestion
{% set pageHeaderTitle = 'Add a category group' %}
{% set pageTitle = applicationName + " - Category groups - " + pageHeaderTitle %}
```

For consistency - we have't included the word 'prisoner' in the other group labels